tests/testthat/test-epi_spread.R

```diff
@@ -0,0 +1,61 @@
+# basic test that solver returns numerics within range
```

Comment:
I think line is residual from another file, should be deleted?

Reply:
Thanks, removed in https://github.com/epiverse-trace/finalsize/pull/42/commits/b2f80d388bd22eefde8787c91bc24c55658c115e